src/Cool/CollaboraDiscoveryFetcher.php

```diff
@@ -90,6 +90,8 @@ protected function getDiscoveryUrl(): string {
       throw new CollaboraNotAvailableException('The configured Collabora Online server address is empty.');
     }
     $wopi_client_server = trim($wopi_client_server);
+    // The trailing slash in the configured URL is optional.
+    $wopi_client_server = rtrim($wopi_client_server, '/');
```

Comment:
Technically, a URL with double slash like `http://example.com//hello` still works.
But explicitly calling `rtrim(*, '/')` here clarifies the format we expect to find in the config value.

Reply:
We could add tests to verify that it works with and without trailing slash, but I don't really want to.
TBD.